src/Chainweb/Pact/Backend/Utils.hs

```diff
@@ -82,6 +82,7 @@ withSavepoint name action = mask $ \resetMask -> do
         liftIO $ evaluate r
     throwErr s = internalError $ "withSavepoint (" <> asString name <> "): " <> pack s
     handlers = [ Handler $ \(e :: PactException) -> throwErr (show e)
+               , Handler $ \(e :: AsyncException) -> throwM e
```

Comment:
@slpopejoy @gregorycollins should sign off on this

Reply:
Looks good to me